### DISCUSSION TITLE
Github pages - url links with special charters for example _basics.html

### DISCUSSION THREAD

**Discussion by Juuso1** [quarto-dev/quarto-cli#3960] [Q&A]
Hi,

I noticed when I publish my Quarto book on Github pages that if navigation html page starts like this _basics.html or like this basic.html it makes difference. If url link is _basic.html I get 404 error. If link starts like this basic.html it works well? 

BR;
Juuso

**Comment by cderv** [comment]
This is because Github Pages uses Jekyll by default to render static website, and path with `_` are ignored. 

See our doc (https://quarto.org/docs/publishing/github-pages.html#render-to-docs) where we explained [about `.nojekyll`](https://github.blog/2009-12-29-bypassing-jekyll-on-github-pages/)
> Then, add a .nojekyll file to the root of your repository that tells GitHub Pages not to do additional processing of your published site using Jekyll (the GitHub default site generation tool):

Do you have already created this file in your root repo ?

**Comment by Juuso1** [comment]
Ok, thank you for this answer.

Br, juuso
